config/initializers/money.rb

```diff
@@ -1,9 +1,9 @@
 require 'three_scale/money_conversions'
 
-ActiveSupport.on_load(:active_record) do
-  ActiveRecord::Base.send(:include, ThreeScale::HasMoney)
-end
-
 Rails.application.config.to_prepare do
   ActionController::Base.helper(ThreeScale::MoneyHelper)
+
+  ActiveSupport.on_load(:active_record) do
+    ActiveRecord::Base.include ThreeScale::HasMoney
+  end
```

Comment:
:open_mouth: this might have been an overlook previously causing ActiveRecord to be loaded prematurely :thinking:

Reply:
Why? The code that references `ActiveRecord` was not being executed until `ActiveSupport` was loaded

Reply:
```
$ RAILS_ENV=test bundle exec rake ci:db:ready db:create
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
Connected to mysql2://root@127.0.0.1/systemdb0
rake aborted!
NameError: uninitialized constant ThreeScale::HasMoney

  ActiveRecord::Base.include ThreeScale::HasMoney
                                       ^^^^^^^^^^
Did you mean?  Hashery
/home/dmayorov/Projects/3scale/porta/config/initializers/money.rb:4:in `block in <main>'
/home/dmayorov/Projects/3scale/porta/config/initializers/money.rb:3:in `<main>'
/home/dmayorov/Projects/3scale/porta/config/environment.rb:8:in `<main>'
/home/dmayorov/.asdf/installs/ruby/3.1.5/bin/bundle:25:in `load'
/home/dmayorov/.asdf/installs/ruby/3.1.5/bin/bundle:25:in `<main>'
Tasks: TOP => db:create => db:load_config => environment
(See full trace by running task with --trace)

```